receptorctl/receptorctl/cli.py

```diff
@@ -31,14 +31,28 @@ def parse_args(self, ctx, args):
 @click.pass_context
 @click.option('--socket', envvar='RECEPTORCTL_SOCKET', required=True, show_envvar=True,
               help="Control socket address to connect to Receptor (defaults to Unix socket, use tcp:// for TCP socket)")
-def cli(ctx, socket):
+@click.option('--yaml', default=None, envvar='RECEPTORCTL_YAML', required=False, show_envvar=True,
+              help="Yaml filename configured for receptor")
+@click.option('--tlsclient', default=None, envvar='RECEPTORCTL_TLSCLIENT', required=False, show_envvar=True,
```

Comment:
Maybe call this tls-client instead of tlsclient, for the same reason.

Reply:
fixed